### PR TITLE
improves ch08

### PR DESCRIPTION
**Pull request by Nowosad**

### CODE REVIEW

File: 08-read-write-plot.Rmd [hunk 1]
```diff
@@ -22,13 +22,13 @@ Geographic data *input* is essential for geocomputation\index{geocomputation}: r
...
 
-There are many geographic file formats, each with their own advantages and disadvantages, as described in Section \@ref(file-formats).
+There are many geographic file formats, each with its own advantages and disadvantages, as described in Section \@ref(file-formats).
 Reading and writing from and to these file formats is covered in Sections \@ref(data-input) and \@ref(data-output), respectively.
 In terms of where to *find* data, Section \@ref(retrieving-data) describes *geoportals* and how to import data from them.
 Dedicated packages to ease geographic data import, from sources including OpenStreetMap, are described in Section \@ref(geographic-data-packages).
-If you want to put your data 'into production' in web services (or if you want be sure that your data adheres to established standards), geographic metadata is important, as described in Section \@ref(geographic-metadata).
+If you want to put your data 'into production' in web services (or if you want to be sure that your data adheres to established standards), geographic metadata is important, as described in Section \@ref(geographic-metadata).
```
Comment (Robinlovelace) [inline 08-read-write-plot.Rmd:31]: Definite typo fix. They are good!

File: _08-ex.Rmd [hunk 2]
```diff
@@ -50,7 +50,7 @@ plot(germany_borders)
 st_write(germany_borders, "germany_borders.gpkg")
 ```
 
-E5. Download the global monthly minimum temperature with a spatial resolution of five minutes using the **geodata** package.
+E5. Download the global monthly minimum temperature with a spatial resolution of 5 minutes using the **geodata** package.
```
Comment (Robinlovelace) [inline _08-ex.Rmd:53]: Why this? Not consistent with previous change from 8 to eight, right?

**Review by Robinlovelace — APPROVED** [review]
Good stuff, keep them coming!

File: _08-ex.Rmd (hunk 2 above)
Comment (Nowosad) [reply _08-ex.Rmd:53]: This is the case I mentioned to you last week in Prague. I have zero idea why...

Comment (Robinlovelace) [reply _08-ex.Rmd:53]: :shrug:

Comment (Robinlovelace) [reply _08-ex.Rmd:53]: No big deal either way... "5" is shorter so would advocate for that throughout. Not pedantic enough to care, that much...

**Pull request merged**